pull_into_place/big_jobs/pip_design.py

```diff
@@ -34,6 +34,9 @@
             'cst_file=' + workspace.restraints_path,
             'loop_start=' + str(workspace.loop_boundaries[0]),
             'loop_end=' + str(workspace.loop_boundaries[1]),
+            'outputs_folder=' + workspace.output_dir + "/sequence_profiles",
+            'design_number=' + bb_model '_{0:03}'.format(design_id),
```

Comment:
This looks like a syntax error...?

Reply:
Oops, only tested the build step so far!